cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -83,6 +83,20 @@ func GetTraceStoreFactory(name string, host component.Host) (tracestore.Factory,
 	return v1adapter.NewFactory(f), nil
 }
 
+func GetSamplingStoreFactory(name string, host component.Host) (storage.SamplingStoreFactory, bool) {
```

Comment:
I would prefer to return error and let the caller log it. Otherwise how would we catch config errors when say remote sampling processor asks for a storage by the wrong name.